projects/demo/src/modules/directives/sidebar/examples/2/index.html

```diff
@@ -0,0 +1,64 @@
+<div
+    tuiSidebarPortal
+    class="container"
+    [class.sidebar-side]="open && mode.value === 'side'"
+>
+    <tui-sidebar-portal-host></tui-sidebar-portal-host>
+
+    <div class="group">
+        <div tuiGroup>
+            <tui-select [formControl]="mode">
+                Sidenav mode
+                <input
+                    tuiTextfield
+                    placeholder="Choose mode"
+                />
+                <tui-data-list-wrapper
+                    *tuiDataList
+                    [items]="modeState"
+                ></tui-data-list-wrapper>
+            </tui-select>
+
+            <tui-select [formControl]="backdrop">
+                Has backdrop
```

Comment:
nitpicking:

I think selector with true/false values should be replaced with [Toggle](https://taiga-ui.dev/components/toggle) / [Checkbox](https://taiga-ui.dev/components/checkbox).